### PR TITLE
Copyedit image classification neural networks

### PR DESCRIPTION
**Pull request by anisa-hawes**
Hello @nabsiddiqui and @scottkleinman,

I've prepared the copyedits + generated perma.cc links for this lesson. You can [review the changes I've made in the rich diff by navigating to [the "Files changed" tab (above)](https://github.com/programminghistorian/ph-submissions/pull/543/files).

Please let me know if you're happy with the adjustments? You'll notice that I have left a few comments, indicating where I have questions, and where additions or clarifications are needed.

Thank you,
Anisa

### CODE REVIEW

File: en/drafts/originals/image-classification-neural-networks.md [hunk 1]
```diff
@@ -25,150 +25,154 @@ doi: tbc
...
 
-{% include figure.html filename="cat.jpg" caption="Figure 1. A picture of a cat" %}
-{% include figure.html filename="dog.jpg" caption="Figure 2. A picture of a dog" %}
+{% include figure.html filename="cat.jpg" alt="Visual description of figure image" caption="Figure 1. A picture of a cat" %}
```
Comment (anisa-hawes) [inline en/drafts/originals/image-classification-neural-networks.md:58]: As part of our commitment to accessibility, we are now asking authors to provide descriptive alt-text to accompany figure images.

Please slot the alt-text into the template I've created throughout the lesson: `alt="Visual description of figure image"`.

I've collated [some resources on our Wiki](https://github.com/programminghistorian/jekyll/wiki/Achieving-Accessibility-Alt-text-Colour-Contrast#alt-text) as guidance if you need it.

File: en/drafts/originals/image-classification-neural-networks.md [hunk 2]
```diff
@@ -25,150 +25,154 @@ doi: tbc
...
+To begin, create a new folder called `projects`. This folder will hold all relevant files and images. To train the neural network in [Google's Teachable Machine](https://teachablemachine.withgoogle.com/), you will need a collection of labeled images as most neural networks are geared towards what is known as 'supervised learning'. Machine learning can be divided into two forms: supervised and unsupervised learning. Supervised learning involves data that is already labeled. Unsupervised learning, on the other hand, involves identifying patterns and grouping data that is alike together. You may have seen the use of some unsupervised machine learning algorithms such as [K-Means Clustering](https://perma.cc/39XM-PBAS) and [Latent Dirichlet Allocation](https://perma.cc/JM3P-BZWN) in digital humanities research. 
 
 For this tutorial, we will download a dataset of paintings from [ArtUK](https://artuk.org/), which provides access to works that meet the UK's requirements for "public ownership." Before its launch, most of the UK's paintings in public ownership were not in regular circulation. ArtUK combats this by providing the general public access to these materials.
```
Comment (anisa-hawes) [inline en/drafts/originals/image-classification-neural-networks.md:50]: You write that before the launch of ArtUK "most of the UK's paintings in public ownership were not in regular circulation".  I don't think this is true ! For example, the extensive paintings collections of the National Gallery, Tate and the V&A (among others) have long been in circulation. Providing the general public with access to their collections has always been fundamental to the very existence of these institutions !

I'm also confused by the phrasing: "works that meet the UK's requirements for 'public ownership'". 
Is this a quote  ?  What is meant by this ?

File: en/drafts/originals/image-classification-neural-networks.md [hunk 3]
```diff
@@ -25,150 +25,154 @@ doi: tbc
...
+Activation functions are particularly important in the final layer as they constrain the output to within a certain range. If you are familiar with [logistic regression](/en/lessons/logistic-regression), you may be familiar with the [sigmoid function](https://perma.cc/3J3M-FPJQ) which is used in binary classification. We can use this same function as an activation function for a neural network to constrain our values set to `0` or `1`. However, because we normally have more than two categories, the [ArgMax](link) and [SoftMax](link) functions are more common. The former outputs the category with the maximum probability to 1 and the remainder to zero. The latter provides values for each category between `0` and `1` with the highest value indicating the most likely classification and the lowest value indicating the least likely classification.
```
Comment (anisa-hawes) [inline en/drafts/originals/image-classification-neural-networks.md:93]: Please could you provide links to define the categories ArgMax and SoftMax?

File: en/drafts/originals/image-classification-neural-networks.md [hunk 4]
```diff
@@ -25,150 +25,154 @@ doi: tbc
...
 
-Finally, a "pooling layer" is utilized. A pooling layer works similarly to a convolutional layer in that it takes a grid, usually 2x2, and passes it over each value in the feature map. In contrast with the convolution layer, however, the pooling layer simply takes the max or average value of the numbers in the grid. This creates a smaller feature map. Together, convolutions and pooling allow neural networks to do image classification even if the spatial arrangement of the pixels is different and without having to do as many calculations.
+Finally, a "pooling layer" is utilized. A pooling layer works similarly to a convolutional layer in that it takes a grid; usually 2x2, and passes it over each value in the feature map. In contrast with the convolution layer, however, the pooling layer simply takes the maximum or average value of the numbers in the grid. This creates a smaller feature map. Together, convolutions and pooling allow neural networks to perform image classification even if the spatial arrangement of the pixels is different and without having to do as many calculations.
```
Comment (anisa-hawes) [inline en/drafts/originals/image-classification-neural-networks.md:103]: Q: maximum or average ? 

This is confusing to me. Which of these two values (the maximum or the average) ?

File: en/drafts/originals/image-classification-neural-networks.md [hunk 5]
```diff
@@ -25,150 +25,154 @@ doi: tbc
...
 
 As humans, we can easily determine which one is a dog and which one is a cat. However, outlining the exact differences proves challenging. It turns out that humans are usually a lot better at handling the nuances of vision than computers. What if we could get a computer to process images the way our brains do? This question and insight forms the basis of artificial neurons.
 
-As the name implies, artificial neurons take their inspiration from neurons in the brain. The following is a simplified look at what a biological and artificial neuron looks like:
+As the name implies, artificial neurons take their inspiration from neurons in the brain. The following is a simplified diagram[^3] of how a biological and artificial neuron work:
 
-{% include figure.html filename="neuron.png" caption="Figure 3. A diagram of a biological and an artificial neuron. https://www.oreilly.com/library/view/mobile-artificial-intelligence/9781789344073/5ac86b1a-c080-49ea-a234-5335f12f15af.xhtml" %}
+{% include figure.html filename="neuron.png" alt="Visual description of figure image" caption="Figure 3. A diagram of a biological and an artificial neuron." %}
```
Comment (anisa-hawes) [inline en/drafts/originals/image-classification-neural-networks.md:65]: There is no Figure 4 or 5 ? 

We move directly from Figure 3 at l.65 to Figure 6 at l.123.

Are they missing? Or do we need to re-number and re-name remaining images?

File: en/drafts/originals/image-classification-neural-networks.md [hunk 6]
```diff
@@ -25,150 +25,154 @@ doi: tbc
...
+Now, we can begin uploading image samples for each class. You will find that you can either "Upload" images or use your webcam to create new ones. We will be uploading all the images for each of our categories to the training folder.
 
 Under "Class 1", click "Choose images from your files, or drag & drop here." Select the "aircraft" folder from inside the dataset "training" folder and drag it into the Teachable Machine window. Click the pencil icon next to "Class 1" and change the name to "aircraft".
 
 Repeat this process for the other folders in the dataset. After the second time, you will need to click "+ Add a class" for each new folder.
 
-{% include figure.html filename="add_classes.png" caption="Figure 6. Adding classes to Google Teachable Machine." %}
+{% include figure.html filename="add_classes.png" alt="Visual description of figure image" caption="Figure 6. Adding classes to Google Teachable Machine." %}
```
Comment (anisa-hawes) [inline en/drafts/originals/image-classification-neural-networks.md:123]: There is no Figure 4 or 5 ? 

We move directly from Figure 3 at l.65 to Figure 6 at l.123.

Are they missing? Or do we need to re-number and re-name remaining images?

File: en/drafts/originals/image-classification-neural-networks.md (hunk 2 above)
Comment (anisa-hawes) [inline en/drafts/originals/image-classification-neural-networks.md:50]: Line 50.

You write that before the launch of ArtUK "most of the UK's paintings in public ownership were not in regular circulation". I don't think this is true ! For example, the extensive paintings collections of the National Gallery, Tate and the V&A (among others) have long been in circulation. Providing the general public with access to their collections has always been fundamental to the very existence of these institutions !

I'm also confused by the phrasing: "works that meet the UK's requirements for 'public ownership'".
Is this a quote ? What is meant by this ?

**Comment by nabsiddiqui** [comment]
Got it, I will have this done sometime early next week.

File: en/drafts/originals/image-classification-neural-networks.md (hunk 1 above)
Comment (nabsiddiqui) [reply en/drafts/originals/image-classification-neural-networks.md:58]: Will do.

File: en/drafts/originals/image-classification-neural-networks.md (hunk 4 above)
Comment (nabsiddiqui) [reply en/drafts/originals/image-classification-neural-networks.md:103]: It depends on the CNN. Some do maximum while others do averaging so the way it is written is correct.

File: en/drafts/originals/image-classification-neural-networks.md (hunk 5 above)
Comment (nabsiddiqui) [reply en/drafts/originals/image-classification-neural-networks.md:65]: Hmm...this may be due to something I deleted. I will double check real quick.

File: en/drafts/originals/image-classification-neural-networks.md (hunk 2 above)
Comment (nabsiddiqui) [reply en/drafts/originals/image-classification-neural-networks.md:50]: I will clarify this. I believe that is how they pitched it on their website. I will double check.

File: en/drafts/originals/image-classification-neural-networks.md (hunk 4 above)
Comment (anisa-hawes) [reply en/drafts/originals/image-classification-neural-networks.md:103]: Thank you, @nabsiddiqui. I would suggest adding this in, within brackets perhaps. For example:

[...] the pooling layer simply takes the maximum or average value (depending on which CNN you are using) of the numbers in the grid [...]

I can apply this edit.

Comment (nabsiddiqui) [reply en/drafts/originals/image-classification-neural-networks.md:103]: Sure, if you want to add that, I am ok with that. Let me know if you need anything else on my end.

File: en/drafts/originals/image-classification-neural-networks.md (hunk 5 above)
Comment (anisa-hawes) [reply en/drafts/originals/image-classification-neural-networks.md:65]: Thank you for adding these in, @nabsiddiqui.
